packages/functional-tests/tests/subscription-tests/coupons-ui-tests/resubscription.spec.ts

```diff
@@ -148,8 +145,7 @@ test.describe('severity-2 #smoke', () => {
       expect(await subscriptionManagement.getCardInfo()).toContain('4444');
     });
 
-    //Disable as paypal sandbox is taking a long time to load
-    /*test('update mode of payment for paypal', async ({
+    test('update mode of payment for paypal', async ({
```

Comment:
This test has been disabled since 2023/05/10, I'm thinking due to instability when executing steps in the PayPal interface? Currently L189 has a `waitForTimeout` that is raising a lint warning.

What is our level of involvement with the PayPal interface? I'm inclined to remove this test because I'm under the impression we are testing a third party, which should be their responsibility. Changes they make could break our CI without warning and not necessarily mean a break in functionality (false negatives).

Reply:
After discussions with team members, my understanding is that when changing payment methods, the boundary between the SubPlat and PayPal systems occurs when users press the 'Change' button.

<img width="960" alt="image" src="https://github.com/mozilla/fxa/assets/3422688/7b4dee6e-65a0-4cc9-b994-5205091c6fd9">

By reducing the scope of the test to verifying that users are re-directed to PayPal when pressing 'Change' we maintain coverage while reducing intermittency risks in the CI.